src/make_watertight/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
-message("")
```

Comment:
Apologies in advance for commenting on such a small change.

The purpose of these lines is to break up the CMake output to make it easier to see what CMake output is coming from where. This `message("")` line appears in CMakeLists.txt files throughout the entire repo. If we don't want these lines anymore, then we should delete them from all the CMake files, not just this one. (I would argue we shouldn't delete them.)

Reply:
Ah, makes sense. Though I think there's something to their comment that it would be even better if these each contained a message indicating their respective directories.
https://github.com/svalinn/DAGMC/pull/660/files#r368664884

All the same, reverting this change now.